src/components/Map/Map.jsx

```diff
@@ -75,6 +81,9 @@ const Map = (props) => {
           attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
           url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
         ></TileLayer>
+        {data && data.places.map((item) => {
+          return <ItemMarker item={item} key={item.address} icon={icon} />;
+        })}
```

Comment:
If this now works, we can remove the useEffect in this component that renders the places.

Reply:
Does it work for you? If yes, we can refactor it later. We need features now

Reply:
I do not know, as I receive runtime errors before it renders the itemmarkers.